### DISCUSSION TITLE
Stuck while trying to generate session string using Heroku Run Console

### DISCUSSION THREAD

**Discussion by ivanivanoff888** [aahnik/tgcf#223] [Q&A] [answered]
Newbie question. So, as the title says, I am stuck on generating session string in Heroku.
The instruction says: "you may generate a session string by running tg-login in an interactive shell".
Nothing is said about where should I find that interactive shell. I am a newbie in Heroku and this speaks nothing to me.

Using search I found this:
"Start an interactive shell
You can run a command, typically scripts and applications that are part of your app, in a one-off dyno using the heroku run command. It can also be used to launch an interactive PHP shell attached to your local terminal for experimenting in your app’s environment:"
So I went to "More" > "Run console", in the appeared console I entered tg-login and pressed "Enter". In the console appeared "Attaching to dyno..." and then nothing changes.

So what is the more detailed guide to obtaining session string for newbies?

**Comment by aahnik** [comment, marked as answer]
To generate a session string from your browser read this 

https://github.com/aahnik/tgcf/wiki/Login-with-a-bot-or-user-account#run-online

The wiki page has been updated

![image](https://user-images.githubusercontent.com/66209958/123539203-200fd580-d756-11eb-85eb-f7d6f25c09ce.png)

> So I went to "More" > "Run console", in the appeared console I entered tg-login and pressed "Enter". 

That is not the way. Not supported. Heroku does not support that for Docker-based dynos.

**Reply by ivanivanoff888** [reply]
Thank you. Repl variant works!
